pkg/networkservice/common/heal/option.go

```diff
@@ -67,8 +67,16 @@ func WithRestoreTimeout(restoreTimeout time.Duration) Option {
 	}
 }
 
+// WithEndpointChange sets if Connection.EndpointName can be changed with monitor updates
+func WithEndpointChange() Option {
```

Comment:
@Bolodya1997 Could you please explain in what cases we plan do not use this option?

Reply:
It is NSC-only option. Do you mean adding such comment in code?

Reply:
I don't quite follow why do we need the option only for nscs.